external/fv3fit/fv3fit/keras/_models/dense.py

```diff
@@ -93,7 +93,7 @@ def train_dense_model(
     train_batches: tf.data.Dataset,
     validation_batches: Optional[tf.data.Dataset],
 ) -> Predictor:
-    return train_column_model(
```

Comment:
If this rename is disruptive we can leave it as before, but this is a more accurate name now and there was actually very little about this function that was column-specific before.